docs/ADL2/master07.11-adl_rules.adoc

```diff
@@ -20,6 +20,24 @@ Each of these is described below.
 
 Assertions can be stated that make explicit intended arithmetic identities among value nodes in an archetype. For example, in an archetype representing {wikipedia}Apgar_score[Apgar Score], five values should sum to equal the total. This can be expressed as the following assertions:
 
+[tabs,sync-group-id=adl-example]
+====
+at-coded ADL2::
++
+--------
+rules
+	$respiratory_effort: Integer := /data[at0002]/events[at0003]/data[at0001]/items[at0009]/value[at9001]/value
```

Comment:
Change tab into 4 spaces. There are a few more (mis)use of tabs in this document.